src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsVolume.ps1

```diff
@@ -51,7 +51,6 @@ Licensed under the MIT License. See License.txt in the project root for license
 
 #>
 function Get-AzsVolume {
```

Comment:
So we won't be providing any backwards compatibility?  Breaking customers right away?

Reply:
Talked to Bala, this is fine.